src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = dailyCost * days;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
Consider using `const` instead of `let` for `totalCost` since its value is not being reassigned, but only modified within the same block scope.